resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/config.scala

```diff
@@ -136,10 +136,16 @@ package object config {
     .createWithDefault(true)
 
   private[spark] val REPORT_INTERVAL = ConfigBuilder("spark.yarn.report.interval")
-    .doc("Interval between reports of the current app status in cluster mode.")
+    .doc("Interval between reports of the current app status.")
     .timeConf(TimeUnit.MILLISECONDS)
     .createWithDefaultString("1s")
 
+  private[spark] val CLIENT_LAUNCH_MONITOR_INTERVAL =
+    ConfigBuilder("spark.yarn.am.launchMonitorInterval")
```

Comment:
sorry missed this in my first pass. One thing here is that normally the spark.yarn.am.  configs are configs that apply to the am, this one is slightly different in that its how often the client pulls the am for status (applies more to the client). We aren't documenting it anyway but perhaps we should name something like spark.yarn.clientAMLaunchMonitorInterval?